packages/js/src/novu.ts

```diff
@@ -39,21 +39,21 @@ export class Novu implements Pick<NovuEventEmitter, 'on'> {
     this.#emitter = new NovuEventEmitter();
     this.#session = new Session(
       {
-        applicationIdentifier: options.applicationIdentifier,
-        subscriberHash: options.subscriberHash,
+        applicationIdentifier: 'applicationIdentifier' in options ? options.applicationIdentifier : undefined,
+        subscriberHash: 'subscriberHash' in options ? options.subscriberHash : undefined,
```

Comment:
```suggestion
        applicationIdentifier: options?.applicationIdentifier,
        subscriberHash: options?.subscriberHash
```

I think could be simplified?

Reply:
this is not possible because of the union we created in the react interface. this is exactly what i meant when i said that we needed to make a lot of hacks to make the react Inbox interface act as we want.
```
export type NovuOptions = KeylessNovuOptions | StandardNovuOptions;
```

where `KeylessNovuOptions` is `{}`

that basically tells ts that in this scenario, options could be an empty object and `applicationIdentifier` is not part of it.

Reply:
i think a found how to make it fix the ts.

Reply:
solved.